src/v2/Components/NavBar/Menus/MobileNavMenu/MobileNavMenu.tsx

```diff
@@ -25,6 +25,7 @@ import {
   NavigatorContextProvider,
   useNavigation,
 } from "./NavigatorContextProvider"
+import { NAV_BAR_BORDER_OFFSET, NAV_BAR_HEIGHT } from "../../constants"
```

Comment:
Had to extract these into their own file to prevent it being undefined because of circular dependencies.

Reply:
#minor - Can we update this `../../` to something absolute?

Reply:
Sure thing — I usually just let VSCode autocomplete — wondering if there's a way to prefer one form or the other.